README.md

```diff
@@ -39,14 +39,8 @@ module Protocols
       end
     end
 
-    defimpl for: Integer do
-      def add(this, other)
-        this + other
-      end
-      def subtract(this, other)
-        this - other
-      end
-    end
+    # delegate `to_s` as is, map `add` and `subtract` to `:+` and `:-` respectively
+    defimpl target: Integer, delegate: :to_s, map: { add: :+, subtract: :- }
```

Comment:
Because I want to have explicit strict double-splatted accepted, and, you know, `unless for.nil?` sucks at parser stage :(